bee-hive/pyproject.toml

```diff
@@ -5,9 +5,9 @@ description = "A multi-agent platform with the vision to facilitate deploy and r
 authors = ["IBM"]
 license = "Apache 2.0"
 readme = "README.md"
+requires-poetry = ">=2.0"
 packages = [
-    { include = "bee_hive", from = "." },
-    { include = "cli", from = "." }
+	{include = "*", from="bee_hive"}
 ]
 
```

Comment:
See line 35 - that command needs updating from beeAI to the new name.
(unable to add comment there as not affected by diff)

Reply:
OK